### ISSUE TITLE
Errors after updating VS Code to 1.25.0

### ISSUE THREAD

**Issue by AndisGrossteins** [Gruntfuggly/todo-tree#48]
After latest VSCode June update (v1.25.0), I'm getting following error in developer console every time I switch between documents:
```
TypeError: Cannot read property 'document' of undefined
	at Timeout.highlight [as _onTimeout] (C:\Users\Andis\.vscode\extensions\gruntfuggly.todo-tree-0.0.64\extension.js:613:36)
	at ontimeout (timers.js:386:14)
	at tryOnTimeout (timers.js:250:5)
	at Timer.listOnTimeout (timers.js:214:5)
```

Multiple other extensions are also throwing similar errors and some misbehave because of this change in VS Code.

**Comment by Gruntfuggly** [comment]
I've just uploaded a new version which should prevent the exception being thrown. If you find that the highlighting no longer works, you can try adjusting the highlight delay.

**Comment by AndisGrossteins** [comment]
Hi,

Thank you for quick response and the fix.

> Does this happen with a particularly large file? Or is your machine heaviliy loaded at the time?
This was happening with all files and no, machine load was normal - (below 10% CPU on all four cores an nominal I/O).

I can't recall if in-source tag highlighting was working at the time, as I was troubleshooting a related issue in another extension.

I've updated todo-tree to v0.0.66. As expected, the errors are gone, and tag highlighting in files works. I also tested the new option by setting it to `1000` and then to `100`. It works in both cases.
So, I'm not sure if the issue was related to changes in the latest VSCode release or something simply didn't work right after upgrade.

It seems that this issue can be closed now.
